.github/workflows/release-trigger.yaml

```diff
@@ -0,0 +1,34 @@
+name: Check Version Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push tag for each updated package
+        run: |
+          git config --global user.name "SDK Releaser Bot"
+          git config --global user.email "noreply@stackit.de"
+
+          for file in $(git diff --name-only HEAD~1..HEAD | grep pyproject.toml); do    
+              if git diff HEAD~1..HEAD $file | grep "version ="; then
+                  version_changes=$(git diff HEAD~1..HEAD $file | grep "version =")
```

Comment:
This has `git diff HEAD~1..HEAD $file | grep "version ="` in two places, right? Couldn't it be simplified?

Reply:
I think these kind of scripts benefit with some more comments explaining what we want to achieve in each section, e.g. that `if` block in line 23

Reply:
I added a more comments. Btw the `if` in line 23 already had a comment.